go.mod

```diff
@@ -66,6 +66,13 @@ require (
 	google.golang.org/grpc v1.49.0
 )
 
+require (
+	github.com/alecthomas/jsonschema v0.0.0-20200530073317-71f438968921
+	github.com/etclabscore/go-openrpc-reflect v0.0.37
+	github.com/open-rpc/meta-schema v0.0.0-20201029221707-1b72ef2ea333
+	golang.org/x/text v0.3.7
+)
```

Comment:
Why is this a separate require? If we want to group those, maybe we should comment what are they related to

Reply:
must have been automatic